src/main/java/collections/FrequencyCounter.java

```diff
@@ -0,0 +1,32 @@
+package collections;
+
+import java.util.TreeMap;
+
+public class FrequencyCounter {
+
+    public static void main(String[] args) {
+
+        String text = "abraCadabra";
+        System.out.println("Text:\n" + text);
+        System.out.println("\nFrequency of characters:\n" + countCharFrequency(text));
+    }
+
+    public static TreeMap countCharFrequency(String text) {
+
+        TreeMap<String, Integer> frequencyDictionary = new TreeMap<String, Integer>();
```

Comment:
it is good practice in Java to have interfaces as a variable types (and assign some class) - e.g. "Map<String, Integer> frequencyDictionary = new TreeMap<String, Integer>();" (first Map, not a TreeMap, but new TreeMap in assignment). the same is for return type of method - should be just Map

Reply:
done

Reply:
TreeMap was changed to a Map in method signature, but there is still a raw use of parameterized class 'Map', please change it to Map<String, Integer>

Reply:
done

Reply:
done